performance_regression_test.py

```diff
@@ -672,3 +677,136 @@ def test_timeseries_bench(self):
+        # self.disable_autocompaction_on_all_nodes()
+        self.run_increasing_by_step_ops_load(stress_cmd_m_templ,
+                                             start_ops, max_ops, throttle_step,
+                                             stress_num, num_loaders, compaction_strategy)
+
+    def test_write_with_stairs(self):  # pylint: disable=too-many-locals
+        """
+        Test steps:
+
+        1. Run a write workload as a preparation
+        2. Run a mixed workload
+        """
+        stress_cmd_w_templ = self.params.get('stress_cmd_w')
+        start_ops = throttle_step = 10_000
+        max_ops = 150_000
+        stress_num = 1
+        num_loaders = len(self.loaders.nodes)
+
+        self.run_fstrim_on_all_db_nodes()
+        # run a write workload as a preparation
+        compaction_strategy = self.params.get('compaction_strategy')
+        self.preload_data(compaction_strategy=compaction_strategy)
+        self.wait_no_compactions_running(n=400, sleep_time=120)
+        # run a mixed workload
+        # create new document in ES with doc_id = test_id + timestamp
+        # allow to correctly save results for future compare
+        # self.create_test_stats(doc_id_with_timestamp=True)
+        self.run_fstrim_on_all_db_nodes()
+        # self.disable_autocompaction_on_all_nodes()
+        self.run_increasing_by_step_ops_load(stress_cmd_w_templ,
+                                             start_ops, max_ops, throttle_step,
+                                             stress_num, num_loaders, compaction_strategy)
+
+    def test_read_with_stairs(self):  # pylint: disable=too-many-locals
+        """
+        Test steps:
+
+        1. Run a write workload as a preparation
+        2. Run a mixed workload
+        """
+        stress_cmd_r_templ = self.params.get('stress_cmd_r')
+        start_ops = throttle_step = 10_000
+        max_ops = 150_000
+        stress_num = 1
+        num_loaders = len(self.loaders.nodes)
+
+        self.run_fstrim_on_all_db_nodes()
+        # run a write workload as a preparation
+        compaction_strategy = self.params.get('compaction_strategy')
+        self.preload_data(compaction_strategy=compaction_strategy)
+        self.wait_no_compactions_running(n=400, sleep_time=120)
+        # run a mixed workload
+        # create new document in ES with doc_id = test_id + timestamp
+        # allow to correctly save results for future compare
+        # self.create_test_stats(doc_id_with_timestamp=True)
+        self.run_fstrim_on_all_db_nodes()
+        # self.disable_autocompaction_on_all_nodes()
+        self.run_increasing_by_step_ops_load(stress_cmd_r_templ,
+                                             start_ops, max_ops, throttle_step,
+                                             stress_num, num_loaders, compaction_strategy)
+
+    # pylint: disable=too-many-arguments,too-many-locals
+    def run_increasing_by_step_ops_load(self, stress_cmd_templ,
```

Comment:
ah, i do see you have renamed the "base" test function... but not the tests themselves